package.json

```diff
@@ -1,11 +1,10 @@
 {
+  "name": "monorepo",
   "license": "Apache-2.0",
-  "private": true,
-  "scripts": {
-    "test": "echo 'Write some tests!'"
-  },
   "workspaces": [
     "packages/*"
   ],
-  "packageManager": "yarn@3.1.0"
+  "scripts": {
+    "test": "yarn install && yarn workspaces foreach run install && yarn workspaces foreach run test"
```

Comment:
As discussed on slack, but for tying it to this PR... this won't work for us. We'll likely need to follow the effort done in OpenShift Console and create a `build.sh` that handles things like this.

Reply:
#10 updated - it now comes with `build.sh` and `test.sh` scripts.

Reply:
:+1: closing this PR then